sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/data/MetricData.java

```diff
@@ -54,22 +69,54 @@
    */
   public abstract InstrumentationLibraryInfo getInstrumentationLibraryInfo();
 
+  /**
+   * Returns the metric descriptor name.
```

Comment:
These javadocs need to be updated, since there is no descriptor any more.

Reply:
done.